server/src/application/application.controller.ts

```diff
@@ -120,4 +120,29 @@ export class ApplicationController {
   ) {
     return this.applicationService.updateBroaderReviewScore(user, applicationId, scoreId, scoreDto);
   }
+
+  // Workshop Score Section
+  @Get('/:applicationId/workshop-scores')
```

Comment:
This can be scores/workshop and for broader review it can be scores/broader. If that's fine!